index.html

```diff
@@ -187,7 +187,7 @@
             <ul class="listing__agent-contacts">
               <li class="listing__agent-contact"><a href="mailto:<%- listItem.agents[0].email %>"><i class="fa fa-envelope listing__envelope-icon"></i></a></li>
               <li class="listing__agent-contact"><a href="<%- listItem.agents[0].address %>"><i class="fa fa-globe listing__globe-icon"></i></a></li>
-              <li class="listing__agent-contact"><a href="tel:<%- listItem.agents[0].phones[0] %>"><i class="fa fa-mobile listing__phone-icon"></i></a></li>
```

Comment:
Had to do this, some agents don't have telephone numbers. We should make this all a bit more robust.